openmc/material.py

```diff
@@ -517,10 +517,24 @@ def add_element(self, element, percent, percent_type='ao', enrichment=None):
             (natural composition).
 
         """
+        
         cv.check_type('nuclide', element, str)
         cv.check_type('percent', percent, Real)
         cv.check_value('percent type', percent_type, {'ao', 'wo'})
 
+        # Make sure element name is just that
+        if not element.isalpha():
+            raise ValueError("Element name should be given by the "
+                             "element's symbol or name, e.g., 'Zr', 'zirconium'")
+
+        # Allow for element identifier to be given as a symbol or name
+        if len(element)>2:
+            el = element.lower()
+            element = openmc.data.ELEMENT_SYMBOL.get(el,"empty")
+            if element == "empty":
+                msg = 'Element name "{}" not recognised'.format(el)
+                raise ValueError(msg)
```

Comment:
I think it would be more direct to check if `el` is in the dictonary rather than using a default value of `"empty"` and checking if we got that default value:

```suggestion
            if el in openmc.data.ELEMENT_SYMBOL:
                element = openmc.data.ELEMENT_SYMBOL[el]
            else:
                msg = 'Element name "{}" not recognised'.format(el)
                raise ValueError(msg)
```

Reply:
A nice middle ground would be to check against `None`, which is returned by default from [`dict.get(key)`](https://docs.python.org/3/library/stdtypes.html?highlight=dict%20get#dict.get), e.g.
```python
element = openmc.data.ELEMENT_SYMBOL.get(el)
if element is None:
    raise ValueError
```

Reply:
Thanks for the suggestions guys - I'm beginning to appreciate just how elegant Python can be!